core/lib/object_store/src/gcs.rs

```diff
@@ -85,22 +85,22 @@ impl GoogleCloudStore {
 
 impl From<AuthError> for ObjectStoreError {
     fn from(err: AuthError) -> Self {
-        let is_transient = matches!(
+        let is_retriable = matches!(
             &err,
-            AuthError::HttpError(err) if is_transient_http_error(err)
+            AuthError::HttpError(err) if is_retriable_http_error(err)
         );
         Self::Initialization {
             source: err.into(),
-            is_transient,
+            is_retriable,
         }
     }
 }
 
-fn is_transient_http_error(err: &reqwest::Error) -> bool {
+fn is_retriable_http_error(err: &reqwest::Error) -> bool {
```

Comment:
`is_transient` is an established terminology used in multiple components across the core. `is_retriable` isn't currently used anywhere.

Could you please revert renaming changes, as they decrease the consistency?

Reply:
Not that big of an issue, so no opinion except for consistency concerns. If you want to rename it, please rename all the entries.